lib/FullScreenView/FullScreenView.js

```diff
@@ -32,24 +28,31 @@ export const FullScreenView = ({
             data-test-header-close-button
             ariaLabel={ariaLabel}
             icon="times"
-            ref={headerCloseButtonRef}
             onClick={onCancel}
           />
         )}
       </FormattedMessage>
     </PaneMenu>
   );
 
+  const renderDefaultHeader = renderProps => (
+    <PaneHeader
+      {...renderProps}
+      paneTitle={paneTitle}
+      actionMenu={actionMenu}
+      firstMenu={firstMenu}
+    />
+  );
+
   return (
     <div
       id={id}
       data-test-full-screen-view
+      {...props}
```

Comment:
I actually found that this wrapped div can be removed at all.

id and data-test-full-screen-view then can be moved to the Pane itself.